lib/puppet/provider/consul_policy/default.rb

```diff
@@ -0,0 +1,190 @@
+require 'json'
+require 'net/http'
+require 'pp'
```

Comment:
Is `pp` actually required or just a leftover from debugging?

Reply:
You are right, dependency has been removed.